.github/workflows/release-please.yml

```diff
@@ -5,10 +5,13 @@ on:
   push:
     branches:
       - master
+      - "release/*"
```

Comment:
hmm so what happens now in the first `release-please` job? does this create separate PRs for each `release/*` branch that we can then merge to create a new versioned release? and if so will they still just be called `aztec-packages-v*`? 
Or does `release-please` ignore non-master branches?

Reply:
It should create a different PR for each "release/*" branch. And the flow for when we merge that PR is in the description.

Reply:
I need to start reading your descriptions before the code 🤦

Reply:
In fairness, I should probably have those descriptions *in the code*.